setup.cfg

```diff
@@ -488,9 +488,6 @@ check_untyped_defs = False
 [mypy-reconcile.test.test_checkpoint]
 check_untyped_defs = False
 
-[mypy-reconcile.test.test_gabi_authorized_users]
-check_untyped_defs = False
-
```

Comment:
Please add 

```
[mypy-reconcile.test.test_gabi_authorized_users]
disallow_untyped_defs = True
disallow_incomplete_defs = True
```

to enforce mypy testing on your tests. There is a [default](https://github.com/app-sre/qontract-reconcile/blob/master/setup.cfg#L470) to skip it for tests.